integration_tests/env.go

```diff
@@ -170,6 +188,7 @@ func InstallCrdsAndOperator(spinNs string, isClusterMode bool, d Defaults, t *te
 
 func (e *TestEnv) InstallCrds(d Defaults, t *testing.T) bool {
 	ApplyManifest("default", d.CRDManifests, e, t)
+	_, _ = RunCommand("rm -rf ~/.kube/http-cache/ && rm -rf ~/.kube/cache/", t)
```

Comment:
Are there any consequences to this command failing?

Reply:
Not necessarily, I'm mainly using it to let `kubectl`  go and check the CRD on the cluster itself instead of the local cache.
I realize that if I don’t do that, kubectl doesn’t recognize the `spinsvc`, `spinnakeraccounts` because it gets just information from the cache.
So, maybe a warning message?